src/polyfills/custom-elements.js

```diff
@@ -937,9 +937,9 @@ export function copyProperties(obj, prototype) {
  * done.
  *
  * @param {!Window} win
- * @param {!Function=} opt_ctor
+ * @param {!Function} ctor
  */
-export function install(win, opt_ctor) {
+export function install(win, ctor) {
```

Comment:
Why not just remove `ctor` parameter?

Reply:
Because we still need to detect if we're in ESM (real classes) or ES5 (transpiled classes).

Reply:
Can be inlined in `install()` which would make the API a bit cleaner (optional).